ksqldb-functional-tests/src/test/resources/query-validation-tests/select.json

```diff
@@ -3,6 +3,27 @@
     "Tests covering general SELECT clause, a.k.a projection, handling"
   ],
   "tests": [
+    {
+      "name": "new query planner test",
```

Comment:
Two things:
1.  This test does not validate that the new query planner is being used, right?  (Sanity check ask; I don't think we should validate that the new query planner being used in functional tests.  Doing that in a unit test may be amusing, but I am not worried about it.)
2.  Does a query plan need to be generated for this?   (Assuming we check it in.  I'd be fine with not having it.)

Reply:
1. Right. I agree that we don't need (and maybe not even can?) verify it. But I am not worried about it either.
2. Not sure if I understand the question? Can you elaborate?

Reply:
Ah, I feel like the `PlannedTestGeneratorTest` fails if I don't run this function: https://github.com/confluentinc/ksql/blob/master/ksqldb-functional-tests/src/test/java/io/confluent/ksql/test/planned/PlannedTestGeneratorTest.java#L28-L40 to generate the historical plans for a given test.

(I think) Basically, if you keep this test, then you may need to generate a plan for it (otherwise CI will fail on that).

Reply:
Ah. Yes. I did not intent to keep this test -- it's just for "showcasing" on the PR.